### PR TITLE
Fix Perl::Critic useless interpolation errors on tests

### PR DESCRIPTION
**Pull request by reosarevok**
Hopefully, if we get rid of as many of these as possible we will actually be able to use Perl::Critic again.

### CODE REVIEW

**Comment by reosarevok** [comment]
@brainzbot, retest this please

**Comment by reosarevok** [comment]
@brainzbot, retest this please

**Review by mwiencek — COMMENTED** [review]
I haven't looked through everything so there is probably more `qq` that can be `q`.  Does Perl::Critic not recognize those as "useless interpolations," or am I missing something?

File: lib/MusicBrainz/Server/Controller/Account.pm [hunk 1]
```diff
@@ -119,7 +119,7 @@ sub verify_email : Path('/verify-email') ForbiddenOnSlaves DenyWhenReadonly
             current_view => 'Node',
             component_path => 'account/EmailVerificationStatus',
             component_props => {
-                message => l('The user with ID \'{user_id}\' could not be found.',
+                message => l(q(The user with ID '{user_id}' could not be found.),
```
[the remark this answers is not in the document]
Comment (reosarevok) [reply lib/MusicBrainz/Server/Controller/Account.pm:122]: AFAICT with a quick test with update_pot.sh, it works fine :)

**Comment by reosarevok** [comment]
I didn't check Perl::Critic after changing " to qq (I only checked Perl::Critic in test files, for now, since those were already a ton of changes and if tests pass we know they were fine). But I've rechecked all the stuff I changed with the avoiding-escaping-quotes commit.

**Comment by reosarevok** [comment]
@brainzbot, retest this please

**Comment by reosarevok** [comment]
@brainzbot, retest this please

**Pull request merged**